README.md

```diff
@@ -51,7 +72,16 @@ the variable pizza inside your method.
 
 Your Answer:
 ```js
+An object holds unordered data in an organized way.
 // write code here
+
+var pizza = {
+  temperature : 70;
```

Comment:
no semicolon needed, need the comma after 70 to seperate properties/methods on an object